### DISCUSSION TITLE
Setting to disable .bak files

### DISCUSSION THREAD

**Discussion by lawrence-laz** [VerifyTests/Verify#817] [Q&A]
Hi, great library! 

I am looking into adopting this into our solution.

There is currently only one friction point in the workflow I am using and hopefully someone could help out here.

When a test fails, a diff tool is opened and after the inspection I just copy paste the **received** contents into **verified** as a way to accept changes. 
However, this is not enough to make the failing test to pass again as the following exception is thrown on a second run:
```
Message: 
VerifyException : Directory: C:\git\my_repo\tests
Delete:
  - TestClassName.TestMethod.verified.txt.bak
Equal:
  - Received: TestClassName.TestMethod.received.txt
    Verified: TestClassName.TestMethod.verified.txt
```
I use version control and don't really see a need for these `.bak` files, but after going through the issues/discussions/docs on the repo I couldn't find a way to disable them.

Is there a way to disable `.bak` file generation, so the workflow with a simple diff tool could be made smoother?

-----
**EDIT:**
I just realized that `.bak` files are being created by my diff tools and not this library. 😅
Stil, I wonder why the library asks me to delete the `.bak` file, for the test to pass?

**Comment by michelschep** [comment]
Thanks for posting this issue. I have the same issue today and was already wondering what was going on (didn't see this one before).
I'm using WinMerge for the diff.
Did you already manage to solve the issue?

**Reply by lawrence-laz** [reply]
> Thanks for posting this issue. I have the same issue today and was already wondering what was going on (didn't see this one before).
> 
> I'm using WinMerge for the diff.
> 
> Did you already manage to solve the issue?

I ended up disabling backup files in WinMerge options and it works fine now.

**Comment by SimonCropp** [comment]
Verify wants to delete the bak file since it matched the convention of a file outputted a test. note that a single test can output many files. wg when a word doc is split into multiple images. so all files matching the test prefix need to be either cleanup up or accepted

i cant see a way to avoid the bak file creation in winmerge via the command line. so it looks like changing the options is the best workaround

**Reply by lawrence-laz** [reply]
I opened [an issue on winmerge repo](https://github.com/WinMerge/winmerge/issues/1804), hopefully a command line argument could be added to make this work out of the box.

**Reply by lawrence-laz** [reply]
Just an update, it seems like there is a way to disable backup generation via the command line (see the the issue on winmerge linked above):
```
"c:\Program Files\WinMerge\WinMergeU.exe" /cfg Backup/EnableFile=0 /inifile "%userprofile%\wm.ini"
```

**Reply by SimonCropp** [reply]
can u try v 20.5.0
